packages/stylelint-plugin/src/utils/cssSyntax.ts

```diff
@@ -34,8 +34,8 @@ export const BpPrefixVariableMap: Record<Exclude<CssSyntax, CssSyntax.OTHER>, st
 };
 
 export const BpVariableImportMap: Record<Exclude<CssSyntax, CssSyntax.OTHER>, string> = {
-    [CssSyntax.SASS]: "~@blueprintjs/core/lib/scss/variables",
-    [CssSyntax.LESS]: "~@blueprintjs/core/lib/less/variables",
+    [CssSyntax.SASS]: "@blueprintjs/core/lib/scss/variables",
```

Comment:
hmm, this party will get a bit tricky because we have a lot of other tooling which still uses node-sass. we'll need to make this import syntax configurable at the lint rule or plugin level somehow.

Reply:
actually, it looks like we already have this configurable for both stylelint rules:

- https://github.com/palantir/blueprint/blob/67c9e2ec477f0c189782234a8d1404af42e8cc20/packages/stylelint-plugin/src/rules/no-prefix-literal.ts#L42
- https://github.com/palantir/blueprint/blob/67c9e2ec477f0c189782234a8d1404af42e8cc20/packages/stylelint-plugin/src/rules/no-color-literal.ts#L43

we should be ok making this change, especially since it's going to go on the 4.x `next` branch